packages/near-membrane-base/src/environment.ts

```diff
@@ -11,19 +11,25 @@ import {
     GetSelectedTarget,
     CallableLinkPointers,
     CallableGetPropertyValuePointer,
+    SupportFlagsField,
 } from './membrane';
 
 const frameGlobalNamesRegExp = /^\d+$/;
 const ShouldTrapMutation = true;
 const ShouldNotTrapMutation = false;
 
+export interface SupportFlagsObject {
```

Comment:
It is a shame that this needs to be exported as well, because it needs to be duplicated into the init method :(